src/Microsoft.ML.Api/TypedCursor.cs

```diff
@@ -253,36 +237,34 @@ public static TypedCursorable<TRow> Create(IHostEnvironment env, IDataView data,
             return new TypedCursorable<TRow>(env, data, ignoreMissingColumns, outSchema);
         }
 
-        private abstract class TypedRowBase
+        private abstract class TypedRowBase : WrappingRow
         {
             protected readonly IChannel Ch;
```

Comment:
>protected readonly IChannel Ch; [](start = 12, length = 31)

Not related to PR, but why we have channel here?
All it does is just work as IExceptionContext. Why it has to be IChannel?